### ISSUE TITLE
MDK fails to generate view contents in Cameo 21x

### ISSUE THREAD

**Issue by sand7000** [Open-MBEE/exec-cameo-mdk#261]
I am aware that Cameo 21x is not tested but my comment that it at least loads seemed interesting to the devs at the tagup so here is the status.

I am using:
MMS Release 4.0.18
View Editor (VE) Release 4.0.2
MDK Release 5.1.3
Cameo 21x

Test Results
* Plugin installs and loads
    * MMS Menus and options visible
    * Logs in to MMS
    * Model Validation runs
    * Org and Repo commit to MMS and are visible in VE
    * Internal Cameo error when recursively generating view contents.
    * Silently fails to manually commit model elements/views to MMS 

Test model attached and error text below:

[test_21.zip](https://github.com/Open-MBEE/mdk/files/11761117/test_21.zip)

```
Thu Jun 15 12:47:17 EDT 2023
Error count: 6
java.lang.NoSuchMethodError: 'java.util.List com.nomagic.uml2.ext.jmi.helpers.StereotypesHelper.getAllProfiles(com.nomagic.magicdraw.core.Project)'
	at gov.nasa.jpl.mbee.mdk.util.GeneratorUtils.getViewpointStereotype(GeneratorUtils.java:315)
	at gov.nasa.jpl.mbee.mdk.util.GeneratorUtils.getViewpointMethod(GeneratorUtils.java:327)
	at gov.nasa.jpl.mbee.mdk.docgen.ViewViewpointValidator.validateView(ViewViewpointValidator.java:129)
	at gov.nasa.jpl.mbee.mdk.docgen.ViewViewpointValidator.validateView(ViewViewpointValidator.java:156)
	at gov.nasa.jpl.mbee.mdk.docgen.ViewViewpointValidator.run(ViewViewpointValidator.java:100)
	at gov.nasa.jpl.mbee.mdk.generator.ViewPresentationGenerator.run(ViewPresentationGenerator.java:127)
	at com.nomagic.task.ProgressTask.execute(ProgressTask.java:25)
	at com.nomagic.task.Task.construct(Task.java:193)
	at com.nomagic.task.SwingWorker$2.call(SwingWorker.java:214)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.nomagic.task.SwingWorker$SwingWorkerFuture.run(SwingWorker.java:601)
	at com.nomagic.task.BackportedThreadPoolExecutor$Worker.a(BackportedThreadPoolExecutor.java:1593)
	at com.nomagic.task.BackportedThreadPoolExecutor$Worker.run(BackportedThreadPoolExecutor.java:1624)
	at com.nomagic.task.PooledThread.run(PooledThread.java:67)

User information:
    Registered to: ADVENTIUM LABS
    Profile: 688697
    Key ID: LKO2022406_55582
System information:
    JVM:
        Name: OpenJDK 64-Bit Server VM;
        Version: 11.0.19;
        Vendor: Ubuntu;
    OS:
        Name: Linux;
        Architecture: amd64;
        Version: 5.15.0-73-generic;
        Locale: English (United States);
MagicDraw Version:
Cameo Enterprise Architecture 2021x Refresh2
Version: Commercial with expiration
Edition: Enterprise
License: Floating
Build: 20211218_0029-81fa0fd0
Registered to: ADVENTIUM LABS
Expires at: May 10, 2024

Cameo Business Modeler 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211217_2348-81fa0fd0
Expires at: May 10, 2024

Cameo Concept Modeler 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211218_0006-be583cc0
Expires at: May 10, 2024

Cameo Data Modeler 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211217_2351-81fa0fd0
Expires at: May 10, 2024

Cameo Requirements Modeler 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211217_2351-81fa0fd0
Expires at: May 10, 2024

SysML 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211218_0004-81fa0fd0
Expires at: May 10, 2024

UAF 2021x Refresh2 Plugin
Version: Evaluation
Build: 20211218_0025-81fa0fd0
Expires at: May 10, 2024

AutoStyler 2021x Refresh2
Strict UML XMI Exporter 2021x Refresh2
TOGAF 2021x Refresh2
Zachman Framework 19.0 SP1

Third Party Plugins:
AADL Bridge 1.12 (Galois)
Model Development Kit 5.1.3 (OpenMBEE)

MagicDraw Environment
Installed at: /home/esandberg/cameo/
Configuration files: /home/esandberg/.cameoea/2021x/
```

**Comment by chenghaozhou123** [comment]
have you solved this problem?

**Comment by Enquier** [comment]
Fixed by #269
